lib/plottr_components/src/components/outline/BeatView.js

```diff
@@ -88,7 +89,9 @@ const BeatViewConnector = (connector) => {
     }
 
     renderManualSort() {
-      if (this.props.beat.autoOutlineSort) return null
+      const { cards, beat } = this.props
+      if (cards[0].isEmpty || cards.length === 1) return
```

Comment:
I don't think that there's an `isEmpty` attribute on cards(?)

Reply:
Were you thinking of the `lodash` function `isEmpty`?

Reply:
@Quiescent the `isEmpty` attribute is only present if the chapter is empty. I added the check because multiple dropzone is showing when dragging a card. The dropzone for CardView and BeatView both are visible.
![image](https://user-images.githubusercontent.com/19387007/129923763-c174fd35-78b8-4d4e-af70-bab50dfba5cf.png)